### DISCUSSION TITLE
Airspaces in branch qt6.5

### DISCUSSION THREAD

**Discussion by markus2marks** [Akaflieg-Freiburg/enroute#344] [General]
Hi,

i installed enroute from the /feature/Qt6.5 branch on my linux computer but i don't see any airspace, is this normal or is something missing on my pc?

**Comment by kebekus** [comment]
@markus2marks Thank you for your report.

Airspaces are not shown because maplibre for Qt has yet to be ported to Qt6.5. 

- https://github.com/maplibre/maplibre-native-qt
- https://github.com/maplibre/maplibre-native-qt/issues/30

The map you see is a placeholder. It will be replaced with a proper map once maplibre becomes available.

I understand that you managed to compile Enroute on your PC or Mac. Congratulations! Few people get that far. Let me know if you are interested in helping with the development. 

I am closing this issue now, but I will read your reply if you post one.

Best wishes,

Stefan.

**Reply by markus2marks** [reply]
Hi Stefan,

Yes, it was very easy to install it on my Ubuntu machine ;) I like this project, I am a glider pilot and I hope that one day we can use it also for gliding. At the moment I use XCSoar, but I don't like it because the graphics lib is very bad and it's very complicated to integrate new things there.

But some basic things in enroute are missing, like airspace warning, remote control and so on and yes, I would like to help.
When I have time at the moment, I try to understand the code, because a lot of new code has been added in the last year.

**Comment by kebekus** [comment]
@markus2marks Thanks for your quick reply, and thanks for offering to help. It would be easiest if we talked personally. If you send me an e-mail, we could arrange an online meeting later this week (or early next week)…
